docs/codeql/codeql-cli/analyzing-databases-with-the-codeql-cli.rst

```diff
@@ -149,12 +171,50 @@ When you create a CodeQL database, the extractor stores diagnostic data in the d
 
 If the analysis found fewer results for standard queries than you expected, review the results of the diagnostic and summary queries to check whether the CodeQL database is likely to be a good representation of the codebase that you want to analyze.
 
+Integrating a CodeQL pack into a Code Scanning workflow
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. pull-quote::
+
+    Note
+
+    The CodeQL package manager is currently in beta and subject to change. During the beta, CodeQL packs are available only in the GitHub Package Registry (GHPR). You must use version 2.5.8 or later of the CodeQL CLI to use the CodeQL package manager.
+
+Using packs inside your Code Scanning setup allows selecting query packs from various sources.
```

Comment:
Are we calling them "CodeQL packs"?

Reply:
Currently, the docs are inconsistent in this regard and a decision hasn't been made.
Personally, I like the proposal to go with _"CodeQL packages"_ which include _"QL packs"_.

Reply:
In order to not lose this once the diff changes, I added the following comment to the main PR thread:
https://github.com/github/codeql/pull/6339#issuecomment-886513565